py3-tox.yaml

```diff
@@ -0,0 +1,99 @@
+package:
+  name: py3-tox
+  version: 4.24.2
+  epoch: 0
+  description: Command line driven CI frontend and development task automation tool
+  copyright:
+    - license: MIT
+  dependencies:
+    provider-priority: 0
+
+vars:
+  pypi-package: tox
+
+data:
+  - name: py-versions
+    items:
+      3.10: "310"
+      3.11: "311"
+      3.12: "312"
+      3.13: "313"
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - py3-supported-build-base-dev
+      - py3-supported-hatch-vcs
+      - py3-supported-hatchling
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/tox-dev/tox
+      tag: ${{package.version}}
+      expected-commit: 05835bfe5db31dfaa71d9fb146602ffcc6b7bfb9
+
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: ${{vars.pypi-package}} installed for python${{range.key}}
+    dependencies:
+      provides:
+        - py3-${{vars.pypi-package}}
+      runtime:
+        - py${{range.key}}-cachetools
+        - py${{range.key}}-colorama
+        - py${{range.key}}-tomli
+        - py${{range.key}}-pluggy
+        - py${{range.key}}-packaging
+        - py${{range.key}}-chardet
+        - py${{range.key}}-platformdirs
+        - py${{range.key}}-filelock
+        - py${{range.key}}-pyproject_api
+        - py${{range.key}}-virtualenv
+      provider-priority: ${{range.value}}
+    pipeline:
+      - uses: py/pip-build-install
+        with:
+          python: python${{range.key}}
+      - runs: |
+          # Subpackage creates tox binary and resulting conflict when adding py3-supported-tox.
+          mv ${{targets.subpkgdir}}/usr/bin/tox ${{targets.subpkgdir}}/usr/bin/tox-${{range.key}}
+      - uses: strip
+    test:
+      pipeline:
+        - uses: python/import
+          with:
+            python: python${{range.key}}
+            import: ${{vars.pypi-package}}
+        - runs: tox-${{range.key}} --version
+
+  - name: py3-supported-${{vars.pypi-package}}
+    description: meta package providing ${{vars.pypi-package}} for supported python versions.
+    dependencies:
+      runtime:
+        - py3.10-${{vars.pypi-package}}
+        - py3.11-${{vars.pypi-package}}
+        - py3.12-${{vars.pypi-package}}
+        - py3.13-${{vars.pypi-package}}
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/bin
+          ln -sf /usr/bin/tox-3.13 ${{targets.subpkgdir}}/usr/bin/tox
```

Comment:
Binaries should stay versioned. I'd omit this

Reply:
In that case, how would we call the `tox` binary? Other packages would fail if there is no such `tox` binary in the `$PATH`.